crates/sncast/src/starknet_commands/script/run.rs

```diff
@@ -292,9 +295,12 @@ pub fn run(
         .assemble_ex(&entry_code, &footer);
 
     // hint processor
-    let mut context = build_context(BlockInfo::default());
+    let mut context = build_context(&ForgeBlockInfo::default().into());
 
-    let mut blockifier_state = CachedState::from(DictStateReader::default());
+    let mut blockifier_state = CachedState::new(
+        DictStateReader::default(),
+        GlobalContractCache::new(GLOBAL_CONTRACT_CACHE_SIZE_FOR_TEST),
```

Comment:
Is it sufficient for us too?

Reply:
yes but still it should be fine since it's used on per-test basis. It's just cache as well so it's not like we're going to run out of cap